docs/opensearch.md

```diff
@@ -2,14 +2,17 @@
 
 #### Add the repository
 `helm repo remove opensearch-operator`
+
 `helm repo add opensearch-operator https://opensearch-project.github.io/opensearch-k8s-operator/`
 
 #### Update the repository
 `helm repo update`
 
-#### Install the Opensearch Operator
+#### Install/upgrade the Opensearch Operator
 `helm install opensearch-operator --create-namespace -n monitoring opensearch-operator/opensearch-operator -f values-develop.yaml`
 
+`helm upgrade opensearch-operator -n monitoring opensearch-operator/opensearch-operator -f values-develop.yaml`
```

Comment:
hey @roman-vi, what is that doing? and is this related to the current dia jitsi upgrade or the package?